src/main/java/com/commercetools/sync/commons/utils/ResourceIdentifierUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.commercetools.sync.commons.utils;
+
+import io.sphere.sdk.models.Referenceable;
+import io.sphere.sdk.models.ResourceIdentifier;
+
+import javax.annotation.Nullable;
+
+import static java.util.Optional.ofNullable;
+
+public final class ResourceIdentifierUtils {
+
+    /**
+     * Given a {@link Referenceable} {@code resource} of the type {@code T}, if it is not null, this method returns
+     * applies the {@link Referenceable#toResourceIdentifier()} method to return it as a {@link ResourceIdentifier} of
+     * the type {@code T}. If it is {@code null}, this method returns {@code null}.
+     *
```

Comment:
Add this  " If it is {@code null}, this method returns {@code null}." to the @return annotation

Reply:
372d681